node/src/auth/Auth.js

```diff
@@ -0,0 +1,58 @@
+const AUTH = WeDeploy.auth('auth-taste.wedeploy.io');
+
+// Data of the user signed in.
+export const currentUser = AUTH.currentUser;
+
+/**
+ * Creates a new account and signs them in on success.
+ */
+export const createAccount = (event) => {
+	event.preventDefault();
+
+	let form = event.target;
+
+	AUTH.createUser({
+		email: form.email.value,
+		name: form.name.value,
+		password: form.password.value
+	})
+		.then(() => {
+			alert('Account successfully created!');
+
+			signIn(event);
+		})
+		.catch(() => {
+			alert('Sign-up failed. Try another email.');
+		});
+
+	return false;
+}
+
+/**
+ * Signs in a user and redirects to the home page.
+ */
+export const signIn = (event) => {
+	event.preventDefault();
+
+	let form = event.target;
+
+	AUTH.signInWithEmailAndPassword(form.email.value, form.password.value)
+		.then(() => {
+			document.location.href = '/';
```

Comment:
Do we always want to navigate home after sign in?  Shouldn't we redirect the user to their previous location?

Reply:
How would we save that? I'll create another issue to handle that.